slippy/tile.go

```diff
@@ -31,8 +31,39 @@ type Tile struct {
 	SRID uint64
 }
 
+func NewTileLatLon(z uint, lat, lon, buffer float64, srid uint64) *Tile {
+	x := Lon2Tile(z, lon)
+	y := Lat2Tile(z, lat)
+
+	return &Tile{
+		z:      z,
+		x:      x,
+		y:      y,
+		Buffer: buffer,
+		SRID:   srid,
+	}
+}
+
 func (t *Tile) ZXY() (uint, uint, uint) { return t.z, t.x, t.y }
 
+func Lat2Tile(zoom uint, lat float64) (y uint) {
```

Comment:
Add reference to https://wiki.openstreetmap.org/wiki/Slippy_map_tilenames#Implementations